scripts/bases/deploy-test-ens.js

```diff
@@ -37,7 +37,7 @@ module.exports = async (
   await logDeploy(factory, { verbose })
   const receipt = await factory.newENS(owner)
 
-  const ensAddr = receipt.logs.filter(l => l.event == 'DeployENS')[0].args.ens
+  const ensAddr = receipt.logs.filter((l) => l.event == 'DeployENS')[0].args.ens
```

Comment:
To reduce the diff, create a .prettierrc which sets the parameters in order to prevent unnecessary changes. In this case, `arrowParens: "avoid"`

https://prettier.io/docs/en/options.html#arrow-function-parentheses